libs/ledger/src/chain/block_coordinator.cpp

```diff
@@ -205,8 +205,8 @@ BlockCoordinator::BlockCoordinator(MainChain &chain, DAGPtr dag,
     }
   });
 
-  // Initialising the BLS library
-  crypto::bls::Init();
+  // Initialising the MCL library
+  bn::initPairing();
```

Comment:
Not sure we need this here for your changes above

Reply:
You are right, I will remove it.